syntax.php

```diff
@@ -70,8 +75,17 @@ function render($mode, Doku_Renderer $renderer, $data) {
                     $data['user'] = $_SERVER['REMOTE_USER'];
                 }
                 $tags = $hlp->findItems(array('tagger' => $data['user']), 'tag', $data['limit']);
+                
                 $renderer->doc .= $hlp->html_cloud($tags, 'tag', array($hlp, 'linkToSearch'), true, true);
 
+                break;
+            case 'tag':
+                $renderer->info['cache'] = false;
+                
+                $pids = $hlp->findItems(array('tag' => $data['tag']), 'pid', $data['limit']);
+
+                $renderer->doc .= $hlp->html_cloud($pids, 'tag', array($hlp, 'linkToPage'), true, true);
```

Comment:
I'm not sure it makes sense to render the result as a cloud. A normal list would probably be more useful?

Reply:
you are right. cloud doens't make sense here. Changed it to a list.